File: poo-2/semana-8/tarea-memento-pattern/Medico.java
public class Medico extends Persona {
  private String especialidad;
  private int costo;

  public Medico(String nombre, String especialidad, int costo) {
    super(nombre);
    this.especialidad = especialidad;
    this.costo = costo;
  }

  public String getEspecialidad() {
    return especialidad;
  }

  public int getCosto() {
    return costo;
  }
}

File: programacion-avanzada-ii/semana-8/medico.py
import persona

Persona = persona.Persona

class Medico(Persona):
  def __init__(self, nombre, apellido, rut, direccion, especialidad):
    Persona.__init__(self, nombre, apellido, rut, direccion)

    self.especialidad = especialidad

  def __str__(self):
    return f"""
nombre: {self.nombre_completo()}
RUT: {self.rut}
dirección: {self.direccion}
especialidad: {self.especialidad}
    """
